src/general/GameWorld.cs

```diff
@@ -290,10 +290,6 @@ public void ReplacePlayerSpeciesWith(Species species, bool deleteOldPlayer = tru
         if (deleteOldPlayer)
         {
             oldPlayer.Obsolete = true;
-
-            // Remove all populations of the old species to make sure it is extinct
-            oldPlayer.Population = 0;
-            Map.RemoveAllPopulationsOf(oldPlayer);
```

Comment:
What's the reasoning behind this change? The species is marked obsolete so all systems will yell really loudly if there is a situation where the species has somehow stuck around and tries to spawn for example.